packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -13,26 +13,49 @@ import {
   QueryBalanceResponse,
 } from '@agoric/cosmic-proto/cosmos/bank/v1beta1/query.js';
 import { Any } from '@agoric/cosmic-proto/google/protobuf/any.js';
-import { AmountShape } from '@agoric/ertp';
+import { AmountShape, PaymentShape, AmountMath } from '@agoric/ertp';
 import { makeTracer } from '@agoric/internal';
 import { UnguardedHelperI } from '@agoric/internal/src/typeGuards.js';
 import { M, prepareExoClassKit } from '@agoric/vat-data';
 import { TopicsRecordShape } from '@agoric/zoe/src/contractSupport/index.js';
+import {
+  depositToSeat,
+  withdrawFromSeat,
+} from '@agoric/zoe/src/contractSupport/zoeHelpers.js';
 import { decodeBase64 } from '@endo/base64';
 import { E } from '@endo/far';
-import { toRequestQueryJson } from '@agoric/cosmic-proto';
-import { ChainAddressShape, CoinShape } from '../typeGuards.js';
+import { toRequestQueryJson, typedJson } from '@agoric/cosmic-proto';
+import { TimeMath } from '@agoric/time';
+import {
+  ChainAddressShape,
+  CoinShape,
+  DepositProposalShape,
+} from '../typeGuards.js';
 
 /**
  * @import {ChainAccount, ChainAddress, ChainAmount, CosmosValidatorAddress, ICQConnection} from '../types.js';
  * @import {RecorderKit, MakeRecorderKit} from '@agoric/zoe/src/contractSupport/recorder.js';
  * @import {Baggage} from '@agoric/swingset-liveslots';
  * @import {AnyJson} from '@agoric/cosmic-proto';
+ * @import {LocalChainAccount} from '@agoric/vats/src/localchain.js';
+ * @import {Payment} from '@agoric/ertp/exported.js';
+ * @import {TimerService, RelativeTimeRecord, TimerBrand} from '@agoric/time';
+ * @import {IBCChannelInfo} from '../types.js';
  */
 
 const trace = makeTracer('StakingAccountHolder');
 
 const { Fail } = assert;
+
+const FIVE_MINUTES_IN_SECONDS = 300n;
+const SECONDS_TO_NANOSECONDS = 1_000_000_000n;
```

Comment:
the 5 is better inline. e.g. what happens when you want `SIX_MINUTES_IN_SECONDS`.


```suggestion
const SECONDS_PER_MINUTE = 60n;
const NANOSECONDS_PER_SECOND = 1_000_000_000n;
```

as we have in https://github.com/Agoric/agoric-sdk/blob/37083e64678683cbbf16faf670dd001e249bb249/packages/inter-protocol/src/proposals/econ-behaviors.js#L20-L24